spec/factories/eras.rb

```diff
@@ -1,9 +1,9 @@
 FactoryBot.define do
   factory :era do
-    user { nil }
-    name { 'MyString' }
+    name { Faker::Hipster.sentence }
     start_date { '2020-12-19 15:37:22' }
```

Comment:
Will wait to use Faker to generate dates until we know how we will store them (DateTime, int, string, etc.)